README.md

```diff
@@ -98,54 +91,38 @@ The 'extras' module is also published as a separate artifact:
 We also provide a [shaded JAR](manual/shaded_jar/)
 to avoid the explicit dependency to Netty.
 
-If you can't use a dependency management tool, a
-[binary tarball](http://downloads.datastax.com/java-driver/cassandra-java-driver-3.10.0.tar.gz)
-is available for download.
-
 ## Compatibility
 
-The Java client driver 3.10.0 ([branch 3.x](https://github.com/datastax/java-driver/tree/3.x)) is compatible with Apache
-Cassandra 2.1, 2.2 and 3.0+ (see [this page](http://docs.datastax.com/en/developer/java-driver/latest/manual/native_protocol/) for
-the most up-to-date compatibility information).
+The Java client driver 3.10.0 ([branch 3.x](https://github.com/scylladb/java-driver/tree/3.x)) is compatible with Apache
+Cassandra 2.1, 2.2 and 3.0+.
 
-UDT and tuple support is available only when using Apache Cassandra 2.1 or higher (see [CQL improvements in Cassandra 2.1](http://www.datastax.com/dev/blog/cql-in-2-1)).
+UDT and tuple support is available only when using Apache Cassandra 2.1 or higher.
 
 Other features are available only when using Apache Cassandra 2.0 or higher (e.g. result set paging,
-[BatchStatement](https://github.com/datastax/java-driver/blob/3.x/driver-core/src/main/java/com/datastax/driver/core/BatchStatement.java),
-[lightweight transactions](http://www.datastax.com/documentation/cql/3.1/cql/cql_using/use_ltweight_transaction_t.html) 
--- see [What's new in Cassandra 2.0](http://www.datastax.com/documentation/cassandra/2.0/cassandra/features/features_key_c.html)). 
-Trying to use these with a cluster running Cassandra 1.2 will result in 
-an [UnsupportedFeatureException](https://github.com/datastax/java-driver/blob/3.x/driver-core/src/main/java/com/datastax/driver/core/exceptions/UnsupportedFeatureException.java) being thrown.
+[lightweight transactions](https://docs.scylladb.com/using-scylla/lwt/))
 
 The java driver supports Java JDK versions 6 and above.
 
-If using _DataStax Enterprise_, the [DataStax Enterprise Java driver][dse-driver] provides 
-more features and better compatibility.
 
-__Disclaimer__: Some _DataStax/DataStax Enterprise_ products might partially work on 
-big-endian systems, but _DataStax_ does not officially support these systems.
+
+__Disclaimer__: Some _Scylla_ products might partially work on 
+big-endian systems, but _Scylla_ does not officially support these systems.
 
 ## Upgrading from previous versions
 
 If you are upgrading from a previous version of the driver, be sure to have a look at
 the [upgrade guide](/upgrade_guide/).
 
-If you are upgrading to _DataStax Enterprise_, use the [DataStax Enterprise Java driver][dse-driver] for more
-features and better compatibility.
+
 
 ## License
-&copy; DataStax, Inc.
+&copy; ScyllaDB, all rights reserved.
```

Comment:
Is it legal to remove copyright like that? After all we're basing our version on their work. I'm not an expert but that sounds dangerous.

Reply:
Probably so, perhaps we should comment it - so it is there but not visible. I don't like the fact we're copying to begin with

Reply:
If you want we can replace it totally. Feel free to prepare our own version of the README.

Reply:
If we do this option, do I overwrite the existing readme, or have a new file called index-readme or similar and just point to it?what do you suggest?

Reply:
@tzach @haaawk  please chime in here

Reply:
Just override IMO.